packages/core-sdk/test/unit/resources/dispute.test.ts

```diff
@@ -122,7 +141,7 @@ describe("Test DisputeClient", () => {
         targetIpId: "0x1daAE3197Bc469Cb97B917aa460a12dD95c6627c",
         cid: "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR",
         targetTag: "tag",
-        bond: 0,
+        bond: 15,
```

Comment:
Can we make this value smaller? so we don't drain the wallet too fast? If not is OK

Reply:
not needed, this is unit test